onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -77,6 +77,17 @@ BasicBackend::BasicBackend(const ONNX_NAMESPACE::ModelProto& model_proto,
   //validate const subgraphs
   if(!openvino_ep::BackendManager::GetGlobalContext().is_wholly_supported_graph) {
     ie_cnn_network_ = CreateCNNNetwork(model_proto, global_context_, subgraph_context_, const_outputs_map_);
+
```

Comment:
@tsocha  This new changes will be now compatible with OV Master but will break the current support. currently we support OpenVINO 2021.3, OpenVINO 2021.2 and OpenVINO 2021.1. we have a release planned this week to support OpenVINO 2021.4.

To support these changes, the next nearest OV Release would be OpenVINO 2022.1.

Please add  a check **#if defined(OPENVINO_2021_1)**  before start of your code.

Reply:
Done